src/screens/account-details.tsx

```diff
@@ -86,8 +89,47 @@ export default function AccountDetails() {
   )
 }
 
+function useImportTransferredTokens() {
+  const { address: accountAddress } = useParams()
+  const { addToken } = useTokensStore()
+  const { network } = useNetworkStore()
+
+  const { data: transfersFrom } = useGetLogs({
+    event: parseAbiItem(
+      'event Transfer(address indexed from, address indexed to, uint256)',
+    ),
+    args: {
+      from: accountAddress as Address,
+    },
+    fromBlock: network.forkBlockNumber,
+    toBlock: 'latest',
+  })
+  const { data: transfersTo } = useGetLogs({
+    event: parseAbiItem(
+      'event Transfer(address indexed from, address indexed to, uint256)',
+    ),
+    args: {
+      to: accountAddress as Address,
+    },
+    fromBlock: network.forkBlockNumber,
+    toBlock: 'latest',
+  })
+
+  if (accountAddress) {
+    ;[
+      ...new Set([
+        ...(transfersFrom?.map((t) => t.address) || []),
+        ...(transfersTo?.map((t) => t.address) || []),
+      ]),
+    ].forEach((addr) => {
+      addToken(addr, accountAddress as Address)
+    })
+  }
+}
+
 function Tokens({ accountAddress }: { accountAddress: Address }) {
   const { tokens } = useTokensStore()
+  useImportTransferredTokens()
```

Comment:
Ideally, this might be better invoked from within a "sync component" in `src/index.ts`, and only called once (see [`SyncBlockNumber`](https://github.com/paradigmxyz/rivet/blob/main/src/index.tsx#L180-L184) for prior art). 

After initial sync, we should then keep track of transferred tokens from within `usePendingBlock`, and make the block range `fromBlock = prevBlock.number`, and `toBlock = block.number` (still using `useGetLogs`).